### PR TITLE
{bio}[intel/2015b] Cufflinks 2.2.1 + Eigen 3.2.7 + SAMtools 0.1.20 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
Cufflinks should now work w/ Boost 1.59.0, cfr. https://github.com/cole-trapnell-lab/cufflinks/issues/3

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/df87e9e7e94198e64793 for a full test report.

File: easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-intel-2015b-Python-2.7.10-Boost-1.59.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+versionsuffix = '-Python-2.7.10'
+dependencies = [
+    ('Boost', '1.59.0', versionsuffix),
+    ('SAMtools', '0.1.20'),
+    ('Eigen', '3.2.7'),
+    ('zlib', '1.2.8'),
+]
+
+configopts = '--enable-intel64 --with-boost=$EBROOTBOOST --with-bam-libdir=${EBROOTSAMTOOLS}/lib'
+preconfigopts = 'CPPFLAGS=-I$EBROOTEIGEN/include'
```
Comment (wpoely86) [inline easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-intel-2015b-Python-2.7.10-Boost-1.59.0.eb]: use `${ }`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/870dd4a5bf3e34f63ae8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b4190107b9cbba477d2e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/266b0a70944600140199 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**